ui/redux/selectors/publish.js

```diff
@@ -57,18 +57,27 @@ export const selectMyClaimForUri = createSelector(
   selectClaimsById,
   selectMyClaimsWithoutChannels,
   ({ editingURI, uri }, isStillEditing, claimsById, myClaims) => {
-    const { channelName: contentName, streamName: claimName } = parseURI(uri);
+    let { channelName: contentName, streamName: claimName } = parseURI(uri);
     const { streamClaimId: editClaimId } = parseURI(editingURI);
 
+    // force to lower case for case insensitive check
+    claimName = claimName.toLowerCase();
+    contentName = contentName.toLowerCase();
+
     // If isStillEditing
     // They clicked "edit" from the file page
     // They haven't changed the channel/name after clicking edit
     // Get the claim so they can edit without re-uploading a new file
     return isStillEditing
       ? claimsById[editClaimId]
-      : myClaims.find((claim) =>
-          !contentName ? claim.name === claimName : claim.name === contentName || claim.name === claimName
-        );
+      : myClaims.find(function (claim) {
```

Comment:
Retain modern `() =>` notation please.  This is part of the reason we didn't bother to update the eslint spacing problem -- we've been updating code to the modern notation.